fineract-provider/src/main/java/org/apache/fineract/infrastructure/configuration/api/GlobalConfigurationApiResource.java

```diff
@@ -76,29 +76,44 @@ public GlobalConfigurationApiResource(final PlatformSecurityContext context,
     @GET
     @Consumes({ MediaType.APPLICATION_JSON })
     @Produces({ MediaType.APPLICATION_JSON })
-    public String retrieveConfiguration(@Context final UriInfo uriInfo,@DefaultValue("false") @QueryParam("survey") final boolean survey) {
+    public String retrieveConfiguration(@Context final UriInfo uriInfo,
```

Comment:
Ok let me explain you, why I needed to add additional parameter, configname in APi.
In Community App, we have added a functionality wherein, only if address configuration is enabled, the address of the clients would be displayed. 
In order to make this functionality work independent of id, I had to modify the api, to get the configuration by name.